components/bl-worldClock-component/README.md

```diff
@@ -0,0 +1,167 @@
+@bl-customComponent-worldClock-digital-flip-background: rgb(0, 115, 116);
+@bl-customComponent-worldClock-digital-flip-border-radius: 3px;
+@bl-customComponent-worldClock-digital-flip-font-size: 1.5rem;
+@bl-customComponent-worldClock-digital-flip-label-margin: 0;
+@bl-customComponent-worldClock-digital-flip-label-font-size: 1rem;
+@bl-customComponent-worldClock-digital-flip-clock-padding: 10px 5px 0px;
+@bl-customComponent-worldClock-digital-flip-container-width: 34px;
+@bl-customComponent-worldClock-digital-flip-container-height: 48px;
+@bl-customComponent-worldClock-digital-flip-container-perspective-origin: 50% 50%;
+@bl-customComponent-worldClock-digital-flip-container-perspective: 300px;
+@bl-customComponent-worldClock-digital-flip-container-background: rgb(16, 21, 23);
+@bl-customComponent-worldClock-digital-flip-container-border-radius: 3px;
+@bl-customComponent-worldClock-digital-flip-container-box-shadow: 0px 10px 10px -10px rgb(255, 253, 255);
+@bl-customComponent-worldClock-digital-flip-container-margin: 0 3px;
+@bl-customComponent-worldClock-digital-flip-ampm-font-size: 1.1rem;
+@bl-customComponent-worldClock-digital-flip-card-width: 100%;
+@bl-customComponent-worldClock-digital-flip-card-height: 50%;
+@bl-customComponent-worldClock-digital-flip-card-border: 1px solid whitesmoke;
+@bl-customComponent-worldClock-digital-flip-card-span-color: rgb(255, 253, 255);
+@bl-customComponent-worldClock-digital-flip-card-sub-border: 0.1px solid rgba(255, 253, 255, 0.5);
+@bl-customComponent-worldClock-digital-flip-card-border-radius: 3px;
+@bl-customComponent-worldClock-digital-flip-upp-card-transform: translateY(50%);
+@bl-customComponent-worldClock-digital-flip-low-card-transform: translateY(-50%);
+@bl-customComponent-worldClock-digital-flip-unfold-transform-origin: 50% 0%;
+@bl-customComponent-worldClock-digital-flip-unfold-transform: rotateX(180deg);
+@bl-customComponent-worldClock-digital-flip-unfold-animation: unfold 1s cubic-bezier(0.455, 0.03, 0.515, 0.955) 0s 1 normal forwards;
+@bl-customComponent-worldClock-digital-flip-fold-transform-origin: 50% 100%;
+@bl-customComponent-worldClock-digital-flip-fold-transform: rotateX(0deg);
+@bl-customComponent-worldClock-digital-flip-fold-animation: fold 1s cubic-bezier(0.455, 0.03, 0.515, 0.955) 0s 1 normal forwards;
+@bl-customComponent-worldClock-digital-flip-semicolon-width: 4px;
+@bl-customComponent-worldClock-digital-flip-semicolon-height: 48px;
+@bl-customComponent-worldClock-digital-flip-semicolon-border-radius: 50%;
+@bl-customComponent-worldClock-digital-flip-semicolon-span-height: 4px;
+@bl-customComponent-worldClock-digital-flip-semicolon-color: rgb(255, 253, 255);
+````
+
+## Examples
+
+Below is a Codeless Example highlighting how to use the World Clock component:
+
+<p align="center">
+  <img src="./example-images/settings.png" alt="settings" width="500"/>
+</p>
+
+<p align="center">
+  <img src="./example-images/component-preview.png" alt="component preview" width="500"/>
+</p>
```

Comment:
Unfortunately, this guide doesn't show anything that could help.

I'm not talking about removing it, but if you wan't to leave that - make properties image smaller, and add subheading for example

```
## Examples

### Usage
Below is a Codeless Example highlighting how to use the World Clock component:
...

```